audit/audit-gcp.sh

```diff
@@ -2,20 +2,32 @@
 # set -x -e
 CNCF_GCP_ORG=758905017065
 
+SOLEPROJECT=""
+if [ $# -ge 1 ]
+then
+   SOLEPROJECT="^$1\s"
+fi
+
 # gcloud organizations describe $CNCF_GCP_ORG 2>&1
 # ERROR: (gcloud.organizations.describe)
 # User [hh@ii.coop] does not have permission to access organization []
 
+# Do we need other formats? bq does not support other formats very well
 format=json
-echo "# Auditing CNCF CGP Org: ${CNCF_GCP_ORG} #"
-gcloud iam roles list --organization=$CNCF_GCP_ORG --format=$format \
-       > cncf-org.roles.$format
-gcloud organizations get-iam-policy $CNCF_GCP_ORG --format=$format \
-       > cncf-org.policy.$format
-echo "## Iterating over Projects ##"
+if [ ! -z ""]
+then
+    echo "# Auditing CNCF CGP Org: ${CNCF_GCP_ORG} #"
+    gcloud iam roles list --organization=$CNCF_GCP_ORG --format=$format \
+        > cncf-org.roles.$format
+    gcloud organizations get-iam-policy $CNCF_GCP_ORG --format=$format \
+        > cncf-org.policy.$format
+    echo "## Iterating over Projects ##"
+else
+    echo "## Generating audit files only for $1 ##"
+fi
 gcloud projects list \
        --filter "parent.id=$CNCF_GCP_ORG" \
-       --format "value(name, projectNumber)" \
+       --format "value(project_id, projectNumber)" | grep "$SOLEPROJECT" \
```

Comment:
why change this?  Not saying it is wrong (I think we always set them the same in fact)